tests/qos/qos_sai_base.py

```diff
@@ -479,24 +544,23 @@ def updateDockerService(host, docker="", action="", service=""):
                     service=service
                 )
             )
+            logger.info("{}ed {}".format(action, service))
```

Comment:
This is will only get printed for stopping services. The statements after the 'yield' are the ones that are going to be executed during teardown

Reply:
Not sure I understood it correctly. "updateDockerService" is called before and after the "yield", both "stopped" and "started" will be logged.

Reply:
even though the updateDockerService is called before and after the yield. In the teardown path, only the statements after the yield will get executed. So the 'started' msg will not get printed.

Reply:
my bad. hadn't looked at it closely. I see your point